tests/entry_point_test.py

```diff
@@ -12,10 +14,23 @@
 if TYPE_CHECKING:
     from collections.abc import Callable
 
-language_definitions = cast(
-    dict[str, dict[str, str]],
-    loads((Path(__file__).parent.parent.resolve() / "sources" / "language_definitions.json").read_text()),
-)
+
+def load_language_definitions() -> dict[str, dict[str, str]]:
+    possible_paths = [
+        Path(__file__).parent.parent.resolve() / "sources" / "language_definitions.json",
+        Path(os.environ.get("PROJECT_ROOT", ".")) / "sources" / "language_definitions.json",
+        Path.cwd() / "sources" / "language_definitions.json",
+    ]
+
+    for path in possible_paths:
+        if path.exists():
+            return cast(dict[str, dict[str, str]], loads(path.read_text()))
+
+    logging.warning("language_definitions.json not found, using SupportedLanguage directly")
```

Comment:
since this is a test file, we should use assertions:

```suggestion
raise AssertionError("language_definitions.json not found, using SupportedLanguage directly")
```